config/perm/events.js

```diff
@@ -9,6 +9,7 @@ module.exports = function () {
       	cmd: 'is_ticketing_admin'
       }]
     }],
+    //  TODO: verify this one, seems meh to me
```

Comment:
It's only used for filling out the form when editing an event. That's probably no reason to lock it down to basic-user though...